requirements_all.txt

```diff
@@ -727,10 +727,12 @@ freebox-api==1.0.1
 # homeassistant.components.free_mobile
 freesms==0.2.0
 
-# homeassistant.components.fritz
 # homeassistant.components.fritzbox_callmonitor
 fritzconnection==1.10.3
```

Comment:
We may need to update all integrations that depend on `fritzconnection`. Normally all integrations should require the same requirement of a library.

Reply:
Yes, I updated manifest.json for both integration. The difference is that fritz also uses QR so it's enabled only there.
Fine for me to enable it for all if this is easier to maintain.

Simone

Reply:
Alligned